src/chapters/setup-intro-data-types/exercises.rst

```diff
@@ -1,2 +1,210 @@
+      :alt: Startup without Project
+
+      Startup without Project
+
+#. Follow the steps that IntelliJ guides you through, accepting all defaults. When prompted to overwrite IntelliJ settings files, confirm that you want to do so.
+
+
+Instructions
+------------
+
+Work on these exercises in the IntelliJ ``java-web-dev-exercises`` project. Create a
+new class for each numbered exercise. You may name the classes whatever you like, but use
+proper :ref:`naming-conventions` and make sure that the file name matches the class name.
+
+Creating a Package and Classes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Here is how to create a new package to store these exercises, and how to
+create new classes within this package:
+
+#. Click on the folder ``src`` in the Project pane, then right-click (or control-click for 
+   some Mac users) and select *New* and then select *Package*.
+
+   .. figure:: figures/newPackage.png
+      :alt: New Package
+
+      New Package
+
+#. Name the package “exercises”.
+
+   .. figure:: figures/namePackage.png
+      :alt: Name Package
+
+      Name Package
+
+#. Right-click/Control-click on the newly created exercises folder. Select *New* and then *Java Class*.
+
+   .. figure:: figures/newClass.png
+      :alt: New Class
+
+      New Class
+
+#. Name this what you will name your class (for example, in the
+   4th exercise below, you might name the class ``Alice``).
+
+   .. figure:: figures/nameClass.png
+      :alt: Name Class
+
+      Name Class
+
+
+   .. note:: 
+      You will be asked whether you want to add this file to Git.
+      Press the “Yes” button.
+   
+   .. figure:: figures/addToGit.png
+      :alt: Add class to Git
+
+      Add class to Git
+
+#. The new class is now created! You can proceed to write code within
+   it. (Don’t forget to write the ``main`` method!)
+
+   .. figure:: figures/newClassCreated.png
+      :alt: Ready to start
+
+      Ready to start
+
+Exercises
+---------
+
+#. **Input/output**: Write a new “Hello, World” program to prompt the
+   user for their name and greet them by name.
+
+   #. Follow steps 3-5 above to create a new Class inside of your exercises.
+   #. Add an import statement at the the top of the file to include Scanner:
+
+      .. sourcecode:: java
+
+         import java.util.Scanner;
+
+   #. Declare a variable of type ``Scanner`` called ``in``:
+
+      .. sourcecode:: java
+
+         Scanner in = new Scanner(System.in);
+
+   #. Add a question to ask the user:
+
+      .. sourcecode:: java
+      
+         System.out.println("Hello, what is your name:");
+
+   #. Create a variable to store the user's response using the ``Scanner``'s ``.nextLine()`` method
+
+      .. sourcecode:: java
+
+         String name = in.nextLine();
+
+   #. Use concatenation to print the greeting:
+
+      .. sourcecode:: java
+
+         System.out.println("Hello " + name);
+   
+   #. Right-click/Control-click the arrow next to your class and run the program.
```

Comment:
Note to ourselves, in the chapter we should note the 3 ways to run the program:
1) Command line,
2) Right-click name in the sidebar,
3) Menu option.